Laud_Misc_610/CGH/senchus_síl_hír.trig

```diff
@@ -0,0 +1,1363 @@
+
+<#Condaid>
+    a foaf:Person;
+    irishRel:nomName "Condaid";
+    rel:childOf <#Ilíach>;
+    owl:sameAs <http://example.com/Rawl_B502/senchas_síl_h_ir_fo_h_érind.trig#ConnaidBuidi>.
+
+<#Mál>
+    a foaf:Person;
+    irishRel:nomName "Mál";
+    rel:childOf <#RossaLette>;
+    owl:sameAs <http://example.com/Rawl_B502/senchas_síl_h_ir_fo_h_érind.trig#Niall>.
+
+<#RossaLette>
+    a foaf:Person;
+    irishRel:genName "Rossa Lette";
+    owl:sameAs <http://example.com/Rawl_B502/senchas_síl_h_ir_fo_h_érind.trig#Let>.
+
+<#Fergus-994218b0>
+    a foaf:Person;
+    irishRel:nomName "Fergus";
+    rel:childOf <#RossaLette>;
+    owl:sameAs <http://example.com/Rawl_B502/senchas_síl_h_ir_fo_h_érind.trig#Fergus-8934bfee>.
+
+<#Illand>
+    a foaf:Person;
+    irishRel:nomName "Illand";
+    rel:childOf <#Fergus-994218b0>.
+
+<#Gergend>
+    a foaf:Person;
+    irishRel:nomName "Gergend";
+    rel:parentOf <#Munremair>;
+    owl:sameAs <http://example.com/Rawl_B502/senchas_síl_h_ir_fo_h_érind.trig#Gerrgeinn>.
+
+<#Munremair>
+    a foaf:Person;
+    irishRel:nomName "Munremair";
+    owl:sameAs <http://example.com/Rawl_B502/senchas_síl_h_ir_fo_h_érind.trig#Muinremuir>.
+
+<#Cass>
+    a foaf:Person;
+    irishRel:nomName "Cass";
+    rel:parentOf <#FachtnaFáthach>;
+    owl:sameAs <http://example.com/Rawl_B502/senchas_síl_h_ir_fo_h_érind.trig#Cass>.
+
+<#FachtnaFáthach>
+    a foaf:Person;
+    irishRel:nomName "Fachtna Fáthach";
+    owl:sameAs <http://example.com/Rawl_B502/senchas_síl_h_ir_fo_h_érind.trig#FachtnaFáthach>.
+
+<#Bricre>
+    a foaf:Person;
+    irishRel:nomName "Bricre";
+    rel:childOf <#Carbad>;
+    owl:sameAs <http://example.com/Rawl_B502/senchas_síl_h_ir_fo_h_érind.trig#BriccneBiltengaid>.
+
+<#Carbad>
+    a foaf:Person;
+    irishRel:genName "Carbad";
+    rel:childOf <#Cass>;
+    owl:sameAs <http://example.com/Rawl_B502/senchas_síl_h_ir_fo_h_érind.trig#Carbad>.
+
+<#Athirne>
+    a foaf:Person;
+    irishRel:nomName "Athirne";
+    rel:childOf <#Athcló>;
+    owl:sameAs <http://example.com/Rawl_B502/senchas_síl_h_ir_fo_h_érind.trig#Aitheirne>.
+
+<#Athcló>
+    a foaf:Person;
+    irishRel:genName "Athcló";
+    owl:sameAs <http://example.com/Rawl_B502/senchas_síl_h_ir_fo_h_érind.trig#Athchló>.
+
+<#ErgeEchbél>
+    a foaf:Person;
+    irishRel:nomName "Erge Echbél";
+    rel:childOf <#Athcló>;
+    owl:sameAs <http://example.com/Rawl_B502/senchas_síl_h_ir_fo_h_érind.trig#EirrgeEchbél>.
+
+<#Ulaid>
+    a irishRel:populationGroup;
+    irishRel:populationGroupName "Ulaid";
+    irishRel:ancestorGroupOf <#DálFíatach>, <#DálAraide>;
+    owl:sameAs <http://example.com/Rawl_B502/section_11.trig#Ulaid>;
+    rdfs:comment "Cóic ríg fichet do hUltaib rogabsat ríge nÉirenn cenmothát na secht ríg rogabsat do Dál Fíatach, dáig is do chloind Óengusa Turbig Temrach & Úgaine Móir do Dáil Fhiatach.".
+
+<#DálFíatach>
+    a irishRel:populationGroup;
```

Comment:
I don't remember if I mentioned this in this file but `irishRel:PopulationGroup` search and replace :)

Reply:
Oh yes. Thanks for the reminder. I hadn't picked up on that suggestion :-D